src/routes/raids/ViewRaid.tsx

```diff
@@ -122,7 +123,7 @@ const $TotalStats = styled.ul`
   margin-bottom: var(--space-3);
 `
 
-const $StatContainer = styled.ol`
+const $StatContainer = styled.div`
```

Comment:
I'm assuming this was done as part of changing the Cards to be a `ul` - once again, I don't understand why this makes sense at all...

Reply:
yes i think so